tests/tests_fabric/test_wrappers.py

```diff
@@ -101,9 +101,9 @@ def __init__(self):
 @pytest.mark.parametrize(
     "precision, input_type, expected_type, accelerator, device_str",
     [
-        pytest.param(32, torch.float16, torch.float32, "gpu", "cuda:0", marks=RunIf(min_cuda_gpus=1)),
+        pytest.param(32, torch.float16, torch.float16, "gpu", "cuda:0", marks=RunIf(min_cuda_gpus=1)),
         pytest.param(32, torch.float32, torch.float32, "gpu", "cuda:0", marks=RunIf(min_cuda_gpus=1)),
-        pytest.param(32, torch.float64, torch.float32, "gpu", "cuda:0", marks=RunIf(min_cuda_gpus=1)),
+        pytest.param(32, torch.float64, torch.float64, "gpu", "cuda:0", marks=RunIf(min_cuda_gpus=1)),
```

Comment:
@justusschock By leaving the base plugin's implementation the identity, these cases change now: We no longer convert the dtype for the default precision 32. Do you agree with this?